.gitattributes

```diff
@@ -1,7 +1,16 @@
 # Force the bash scripts to checkout with Unix line endings, even on Windows
-
 build-* eol=lf
 config eol=lf
 run-* eol=lf
 spec_* eol=lf
 translator-tests eol=lf
+
+# Explicitly mark binary files for Git
+*.bin binary
+
+# Highlight ksy and kst files as YAML on GitHub
+*.kst linguist-language=yaml
+*.ksy linguist-language=yaml
+
+# Mark generated files for GitHub
+spec/** linguist-generated
```

Comment:
I'm somewhat hesitating about this one, but, I guess you're right, ultimately, they all should be generated.

Reply:
The commits that touch the spec directory make it sound like all of the files are generated from the corresponding KST files. Are some of the files there not generated? If so, it might be a good idea to document that somewhere.